tests/test_cli.py

```diff
@@ -197,9 +197,14 @@ def test_log_encoding(self, caplog, monkeypatch):
                 self.exec_cli(command)
 
         # first message should be 'log encoding: <encoding>'
-        match = caplog.records()[0].message.split(':')
+        match = None
+        for record in caplog.records():
```

Comment:
I hit this problem when playing with the root logger.  Basically I enabled DEBUG on root logger, which enabled DEBUG logging for every python library.  In this case, Docker debug messages popped up before the 'log encoding' and the test failed.  I agree this is overkill, however I didn't think we could guarantee that no other 'info' message from say docker would ever pop up first.

Feel free to nak it, I don't mind.  Just wanted to state my original reason.